sql/postgresql/Golang/README_zh.md

```diff
@@ -0,0 +1,66 @@
+# Antlr v4 Postgresql Golang
```

Comment:
We use only English across the entire repository. Please remove files in other languages. It is in low demand and almost nobody is able to understand and fix such files.

Reply:
delete README_zh.md:  https://github.com/antlr/grammars-v4/pull/2751/commits/dacbe9803f1c8d96b0c5fc904d02bc73193ad634

Reply:
Please have a look again